ui/components/ui/icon/preloader/index.js

```diff
@@ -1,3 +1 @@
-import preloader from './preloader-icon.component';
-
-export default preloader;
+export { default } from './preloader-icon.component';
```

Comment:
Updating these import paths fixes an issue with how storybook 7.6 handles imports. I'm not sure how we might update it on the storybook side?